assets/js/app.js

```diff
@@ -0,0 +1,165 @@
+const displayController = (() => {
+  const domBoard = document.getElementById('board');
+  const renderBoard = (board) => {
+    let renderedBoard = '';
+
+    for (let i = 0; i < board.length; i += 1) {
+      for (let j = 0; j < board[i].length; j += 1) {
+        const cellValue = board[i][j];
+        renderedBoard += ` <div class="cell" onclick='game.applyMove(${i},${j})'>${cellValue}</div>`;
+      }
+    }
+    domBoard.innerHTML = renderedBoard;
+  };
+
+  return {
+    renderBoard
+  };
+})();
+
+const gameBoard = ((dController) => {
+  let board = [[' ', ' ', ' '], [' ', ' ', ' '], [' ', ' ', ' ']];
+  const controller = dController;
+  let currentPlayer;
+  let countMoves = 0;
+
+  const setCurrentPlayer = (cPlayer) => {
+    currentPlayer = cPlayer;
+  };
+
+  const clearBoard = () => {
+    board = [[' ', ' ', ' '], [' ', ' ', ' '], [' ', ' ', ' ']];
+    countMoves = 0;
+  };
+  const getBoard = () => {
+    return board;
+  };
+
+  const applyMove = (indx, indy) => {
+    const symbol = currentPlayer.getSymbol();
+    if (board[indx][indy] === '') {
+      board[indx][indy] = symbol;
+      controller.renderBoard(board);
+      countMoves += 1;
+      return true;
+    }
+    alert('Spot taken');
```

Comment:
This should be production code, we shouldn't have `alert`, please remove the `alert`.